presto-parser/src/main/java/com/facebook/presto/sql/parser/AstBuilder.java

```diff
@@ -673,7 +674,73 @@ public Node visitShowColumns(SqlBaseParser.ShowColumnsContext context)
     @Override
     public Node visitShowStats(SqlBaseParser.ShowStatsContext context)
     {
-        return new ShowStats(getLocation(context), getQualifiedName(context.qualifiedName()));
+        return new ShowStats(Optional.of(getLocation(context)), getQualifiedName(context.qualifiedName()));
+    }
+
+    @Override
+    public Node visitShowStatsParametrized(SqlBaseParser.ShowStatsParametrizedContext context)
+    {
+        // The following properties of SELECT subquery are required:
```

Comment:
> SHOW queries does not make to till analysis phase right now

Sure. What I meant was, this should be done in the rewrite. That's where the "semantic" checks for this type of statement happen.